server/rooms.js

```diff
@@ -229,6 +229,7 @@ class BasicRoom {
 			}
 			counter++;
 			buffer += ',' + this.users[i].getIdentity(this.id);
+			if (this.users[i].getStatus()) buffer += `@${this.users[i].getStatus()}`;
```

Comment:
Should status just always be a part of identity?

Reply:
I think there are some cases where you don't want it.

Reply:
Param in `getIdentity`?

`getIdentityWithStatus` function?

Reply:
I like the latter. I don't think we ever want to `getStatus` without also `getIdentity`, anyway.

Reply:
b78af3f adds `getIdentityWithStatus`